docs/self-managed/zeebe-deployment/operations/management-api.md

```diff
@@ -0,0 +1,37 @@
+---
+id: management-api
+title: "Management API"
+description: "Additional cluster management API"
+---
+
+Besides the grpc api for process instance execution, Zeebe Gateway also exposes an HTTP endpoint for cluster management operations. This api is not expected to be used by a normal user, but by a privileged user such as a cluster administrator. It is thus exposed via a different port which is configured using configuration `server.port` (or via environment variable SERVER_PORT). By default, this is set to `9600`. Following operations are available now:
```

Comment:
:wrench: I think it might be worth mentioning that these are spring actuators as that can point users in the right direction to configure security etc.